tesseract-core/src/lib.rs

```diff
@@ -776,6 +781,12 @@ impl Schema {
 
         Ok(column)
     }
+
+    pub fn get_cube_by_name(&self, cube_name: &str) -> Result<&Cube, Error> {
```

Comment:
This is good, I think that I should probably refactor a bunch of methods on `Schema` to be on `Cube` instead. This will help move towards that.